src/main/java/uk/gov/hmcts/reform/sscs/service/HearingsService.java

```diff
@@ -33,6 +37,20 @@ public void processHearingRequest(HearingWrapper wrapper) throws UnhandleableHea
         switch (wrapper.getState()) {
             case CREATE_HEARING:
                 createHearing(wrapper);
+                HearingRequestPayload hearingRequestPayload = new HearingRequestPayload();
+                HearingResponse response =
+                    hearingApi.createHearingRequest("authorisation",
+                                                    "serviceAuthorization",
+                                                    hearingRequestPayload);
+                //Fake payload for now. Ask Lucas about differences in mapping objects.
+                //check which case data to use Ask Lucas
```

Comment:
Use updated, original is there to be able to check for changes before updating the case data.

Reply:
Yeah sweet